### DISCUSSION TITLE
Facefusion

### DISCUSSION THREAD

**Discussion by seoeaa** [lllyasviel/Fooocus#1581] [Ideas]
Proposing to make a window and implement a process of face replacement

https://github.com/facefusion/facefusion

**Comment by khaldi-yass** [comment]
Face swapping is already implemented, check https://www.youtube.com/watch?v=PaGrBsD2aNM

**Reply by ghzgod** [reply]
This is no where near what it should be from the original face.  Reactor and Facefusion are leaps and bounds better than the current implementation.  The thumbnail itself shows 2 different people that look similar.
